sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/PageList.java

```diff
@@ -67,4 +73,24 @@ public PageList setClearRange(List<ClearRange> clearRange) {
         this.clearRange = clearRange;
         return this;
     }
+
+    /**
+     * Get the nextMarker property: The NextMarker property.
+     *
+     * @return the nextMarker value.
+     */
+    public String getNextMarker() {
```

Comment:
is it OK to have this in a model that's returned from not pageable API?

Reply:
@srnagar Can you comment on this? 

I'm not sure how feasible it will be to change as it's a generated type and will likely not take it from me alone that the swagger should be changed as he doesn't like updating .net. I don't know that we can add a whole separate type for the pageable apis through transforms and customizations.

Reply:
We could use the customization feature we have in java to simply make this a private method. It should be straightforward to change the visibility of this generated method. See https://github.com/Azure/autorest.java/blob/main/customization-base/README.md#change-method-modifier

Reply:
We can't make the method private or package private because it's used in one of the clients outside the models package

Reply:
Updated to use the accessor pattern